### ISSUE TITLE
Key error at Telegram build_message

### ISSUE THREAD

**Issue by humrochagf** [rougeth/bottery#24]
When we send something that isn't a message (e.g.: an edit to a sent message) to the bot, it breaks due to a the absence of the key `'message`' at the response.

For example, when we edit a message, the response data contains `'edited_message'` key instead `'message'` which raises an key error at line 87 at `telegram.py` file:

```
 87, in build_message
    id=data['message']['message_id'],
KeyError: 'message'
```

**Comment by humrochagf** [comment]
I've made the PR #25 in order to solve that problem

**Comment by rougeth** [comment]
Closed at #25
